locale/de_DE.json

```
@@ -3451,6 +3451,50 @@
+  "Gestione Collane": "Reihenverwaltung",
+  "Gestisci le collane e le serie di libri": "Buchreihen und Sammlungen verwalten",
+  "Collane totali": "Reihen insgesamt",
+  "Libri nelle collane": "Bücher in Reihen",
+  "Nessuna collana trovata. Aggiungi una collana a un libro per iniziare.": "Keine Reihen gefunden. Fügen Sie einem Buch eine Reihe hinzu.",
+  "Collane": "Reihen",
+  "Serie e collane di libri": "Buchreihen und Sammlungen",
+  "Rinomina collana": "Reihe umbenennen",
+  "Rinomina": "Umbenennen",
+  "Unisci con altra collana": "Mit anderer Reihe zusammenführen",
+  "Nome collana di destinazione": "Name der Zielreihe",
+  "Unisci": "Zusammenführen",
```

Biome (2.4.7)

[error] The key Unisci was already declared.

(lint/suspicious/noDuplicateObjectKeys)